.circleci/config.yml

```diff
@@ -48,9 +49,12 @@ jobs:
               # failures for local file:/// -- yoh found no better way,
               linkchecker -t 1 --check-extern \
                 --ignore-url 'file:///.*' \
-                --ignore-url https://fonts.gstatic.com \
-                --ignore-url "https://github.com/bids-standard/bids-specification/(pull|tree)/.*" \
-                --ignore-url "https://github.com/[^/]*" \
+                --ignore-url 'https://fonts.gstatic.com' \
+                --ignore-url 'https://github.com/bids-standard/bids-specification/(pull|tree)/.*' \
+                --ignore-url 'https://github.com/[^/]*' \
+                --ignore-url 'https://doi.org/.*' \
+                --ignore-url 'https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/10-microscopy.html' \
+                --ignore-url 'https://bids-specification.readthedocs.io/en/stable/99-appendices/14-glossary.html' \
```

Comment:
Why shouldn't we just ignore all `stable` links?

```suggestion
                --ignore-url 'https://bids-specification.readthedocs.io/en/stable/.*' \
```

Reply:
Can't think of a reason.

Reply:
Just to be clear: this means that none of the internal links will be checked then. Am I understanding this correctly?

Reply:
I think that this will only impact the links in the HTML headers, because the internal spec links (in the text) are looking like this:

```html
<p>Additional fields can include external behavioral measures relevant to the
scan.
For example vigilance questionnaire score administered after a resting
state scan.
All such included additional fields SHOULD be documented in an accompanying
<code>_scans.json</code> file that describes these fields in detail
(see <a href="02-common-principles.html#tabular-files">Tabular files</a>).</p>
<p>Example <code>_scans.tsv</code>:</p>
<div class="highlight"><pre><span></span><code>filename    acq_time
func/sub-control01_task-nback_bold.nii.gz   1877-06-15T13:45:30
func/sub-control01_task-motor_bold.nii.gz   1877-06-15T13:55:33
```

that is: They don't match the full form `https://bids-specification.readthedocs.io/en/stable/.*`

Reply:
ah OK 
thanks for the clarification